docs/internal/config/Element-Type-Specs.md

```diff
@@ -42,7 +42,8 @@ These fields are common to all element types and have their own field in the `te
   - default: `{"value": true}`
 - **is_editable**: `JSON` -- dynamic query determining whether can be edited (Would only be false in rare circumstances)
   - default: `{"value": true}`
-- **defaultValue**: `JSON` - the default value for the response. Note: several plugins have their own "default" parameter, which may be easier to use in many situations. What's the difference? In this one, the default response is generated when the application is first created, and from then on it behaves just like a normal question element. The default parameter for individual elements is calculated when the element is rendered, so it can change in response to other changes. If you need a default value to be made from answers the user provided earlier in the form, that would be the one to use. See individual elements for the respective details.  
+- **initialValue**: `JSON` - a dynamic query that will be evaluated to set the initial response value. It's kind of like a "default", but should be thought of as more of a fixed initialisation value (defaults can change in response to other elements). Most plugins have their own "default" parameter, which is probably easier to use in most situations.  
```

Comment:
Should be improved to explain or give an example for when to use it with just passing the basic element response object

Reply:
Okay, done: 21c0769f

I can't actually think of many realistic use cases for `initialValue` -- the ones I've provided here are pretty rare cases. Can you think of any more common ones?